packages/api/src/builder/routes.ts

```diff
@@ -45,6 +55,21 @@ export type Api = {
       HttpStatusCode.SERVICE_UNAVAILABLE
     >
   >;
+  getExpectedWithdrawals(
+    state_id: StateId,
+    proposal_slot?: Slot
+  ): Promise<
+    ApiClientResponse<
+      {
+        [HttpStatusCode.OK]: {
+          executionOptimistic: ExecutionOptimistic;
+          finalized: Finalized;
```

Comment:
Currently Lodestar doesn't support `finalized` in the apis. You can discard this field for now. See #5693 for more.

Reply:
When you remove `finalized` field, don't forget to update `packages/api/test/unit/beacon/oapiSpec.test.ts`. You need to remove `getNextWithdrawals` from `ignoredOperations`, and add `getNextWithdrawals: {response: ["finalized"]},` to `ignoredProperties`